### ISSUE TITLE
There is no class ModalScrollController

### ISSUE THREAD

**Issue by kleyos** [jamesblasco/modal_bottom_sheet#108]
Hey there! I am trying to create a modal with a scrolling.  I use [this](https://github.com/jamesblasco/modal_bottom_sheet/blob/master/example/lib/modals/modal_with_scroll.dart) example.  But there is no such class ModalScrollController. and I can do `controller: ModalScrollController.of(context)`.
Alternatively, I've tried 
```
await showCupertinoModalBottomSheet(
    context: context,
    builder: (context, scrollController) => Subpage(
      controller: scrollController,
      content: child,
    ),
    topRadius: Radius.circular(16),
  );`
```

but it has no effect on the list and scrolling. What I do wrong?

**Comment by jamesblasco** [comment]
You can try the dev version  1.0.1-dev. It has the class you need. 

You should use that scrollController in the param `controller` from the list you are using

**Comment by jamesblasco** [comment]
Now published in version 1.0.0
